### PR TITLE
feat: prepare mainnet

### PR DESCRIPTION
**Pull request by lightscale-luke**
* Set Default ChainID to 255 (main net chain Id)
* Provide shell script for downloading KZG params.
* Disable creating params for KZG. Only allowing loading them.
* Set proper names for params and seed directory.

### CODE REVIEW

File: contrib/download_params.sh [hunk 1]
```diff
@@ -5,10 +5,10 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
 PROJ_DIR=$DIR"/.."
 
 pushd $PROJ_DIR
-mkdir -p test_params
+mkdir -p kzg_params
 RELEASE_VERSION=params-0320
-wget https://circuit-release.s3.us-west-2.amazonaws.com/circuit-release/$RELEASE_VERSION/params20 -O ./test_params/params20
-wget https://circuit-release.s3.us-west-2.amazonaws.com/circuit-release/$RELEASE_VERSION/params26 -O ./test_params/params26
+wget https://circuit-release.s3.us-west-2.amazonaws.com/circuit-release/$RELEASE_VERSION/params20 -O ./kzg_params/params20
```
Comment (dongchangYoo) [inline contrib/download_params.sh]: Our circuits require degrees 21 and 26.

File: zkevm/tests/integration.rs [hunk 2]
```diff
@@ -21,19 +21,19 @@ fn test_load_params() {
     init();
     log::info!("start");
     load_params(
-        "/home/ubuntu/scroll-zkevm/zkevm/test_params",
+        "/home/ubuntu/scroll-zkevm/zkevm/kzg_params",
```
Comment (fakedev9999) [inline zkevm/tests/integration.rs:24]: Sorry, but is this the right path? scroll-zkevm?

**Review by fakedev9999 — APPROVED** [review]
Else, looks good to me!

**Comment by dongchangYoo** [comment]
The seed is a file, not a directory. Please see a commit title of "[refac: change default name of seed directory](https://github.com/kroma-network/kroma-prover/pull/22/commits/0f31524a0ce71b8836338ea8ce54e4e7e3cb1f27)"

File: contrib/download_params.sh (hunk 1 above)
Comment (lightscale-luke) [reply contrib/download_params.sh]: I will remove this file since it's no longer used.

File: zkevm/tests/integration.rs (hunk 2 above)
Comment (lightscale-luke) [reply zkevm/tests/integration.rs:24]: I think this path should be updated as local user's directory environment. 
I changed the last directory name for default directory name

**Comment by dongchangYoo** [comment]
LGTM

File: zkevm/tests/integration.rs (hunk 2 above)
Comment (fakedev9999) [reply zkevm/tests/integration.rs:24]: Oh it is ignored so it was fine. Maybe let's fix the path later when enabling it.

**Review by chokobole — APPROVED** [review]
LGTM

**Review by fakedev9999 — APPROVED** [review]
LGTM

**Comment by dongchangYoo** [comment]
In the "CHAIN_ID=" case, the prover works. I think it is not proper.

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Pull request merged**